uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/absence-tab.js

```diff
@@ -0,0 +1,27 @@
+(function (CRM, require) {
+  CRM.vars.leaveAndAbsences = {
+    baseURL: CRM.absenceTabApp.path
+  };
+
+  var srcPath = CRM.vars.leaveAndAbsences.baseURL + '/js/angular/src/leave-absences';
```

Comment:
Not sure what you mean by backend. For My/manager Leave there is a html page which is used by drupal, and that html defines these variables. In absence tab we dont have a html like that that is why defined the variable in the JS.